README.md

```diff
@@ -1,2 +1,94 @@
-# ui-library
-Elegant design system &amp; Vue.js 3 UI library for moja global
+<div align="center">
+<img src="https://moja.global/wp-content/uploads/2021/03/Asset-66@4x.png" alt="Moja Global UI Library logo" height ="auto" width="200" /> 
+
+<br />
+
+<h1 align="center">UI Library</h1>
+
+ [npm](https://www.npmjs.com/package/@moja-global/mojaglobal-ui)   [yarn](https://yarnpkg.com/package/@moja-global/mojaglobal-ui)
+ </div>
+
+## About The Project
+
+It is an Elegant design system built on Vue.js version 3 for moja global. It is an open-source UI library having pre-written code for various widely used components across all the repositories of moja global that developers and users can use in the projects just by installing the package and running import commands
+ 
+## Built with
+
+- [Vue.js](https://vuejs.org/) version 3
+
+## Components 
+
+- Accordion
+- Alert
+- Button
+- Card
+- Datepicker
+- Dropdown
+- Footer
+- Modal
+- Navbar
+- Slider 
+- Sponsor
+- Toggle
+
+Detailed description about each of the component and how to use the component can be found  [here](https://ui-library-au5vxcs10-moja-global.vercel.app/?path=/story/example-accordion--primary)
```

Comment:
Put this as a table instead showing all the components that are implemented and planned.